server/config/custom-environment-variables.json

```diff
@@ -34,7 +34,8 @@
     "foreign": {
       "apiKey": "COMMUNITY_WEB3_API_KEY",
       "name": "COMMUNITY_WEB3_NETWORK",
-      "gasStation": "COMMUNITY_WEB3_GAS_STATION"
+      "gasStation": "COMMUNITY_WEB3_GAS_STATION",
+      "allowedTokensToRelay": "COMMUNITY_FOREIGN_TOKENS_ALLOWED_TO_RELAY"
```

Comment:
if you do:
```
{
        "__name": "COMMUNITY_FOREIGN_TOKENS_ALLOWED_TO_RELAY",
        "__format": "json"
      }
```

then you can define it as an array: `COMMUNITY_FOREIGN_TOKENS_ALLOWED_TO_RELAY="['0x..']" then you don't need do the split